applications/callflow/src/cf_blacklist.erl

```diff
@@ -0,0 +1,175 @@
+%%%-----------------------------------------------------------------------------
+%%% @copyright (C) 2011-2019, 2600Hz
+%%% @doc
+%%% @author Sergey Safarov sponsored by Audian
+%%% @end
+%%%-----------------------------------------------------------------------------
+-module(cf_blacklist).
+
+-export([lookup/4]).
+
+-include("callflow.hrl").
+
+-record(pattern, {action :: kz_term:ne_binary()
+                 ,caller_name :: kz_term:api_ne_binary()
+                 ,has_groups :: boolean()
```

Comment:
Should either default this to true or false or update type to `boolean() | 'undefined'`.

Reply:
I renamed record name. But modules functions still have name `bl_action`.